.github/workflows/release.yml

```diff
@@ -0,0 +1,28 @@
+name: Release on PyPI
+
+on:
+  push:
+    tags:
+    - v*
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
```

Comment:
Hmm, does this work?  Which Python does it use actually?

If in doubt I'm generally suggesting python 3.8 now, since that's available for RHEL8 and the oldest python3 we'd have to support in most cases.

Reply:
Updated to use Python 3.8